sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/SecHubRemoteCredentialSupportTest.java

```diff
@@ -0,0 +1,181 @@
+package com.mercedesbenz.sechub.commons.model;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.NullSource;
+
+import com.mercedesbenz.sechub.test.TestFileReader;
+
+class SecHubRemoteCredentialSupportTest {
+
+    private SecHubRemoteCredentialSupport supportToTest;
+
+    @BeforeEach
+    void beforeEach() {
+        supportToTest = new SecHubRemoteCredentialSupport();
+    }
+
+    @Test
+    void read_simple_credential_configuration_from_json_string() {
```

Comment:
Looking into the configuration file and this test at the end, "simple" means here it is just one entry. 
```suggestion
    void read_credential_configuration_with_one_entry_only_from_json_string() {
```

"Simple" is ... not so clear at first sigh - please rename the file and also the method name, so it is clear that there is just one entry inside. Would make it easier to understand.